.github/workflows/rich-codex.yml

```diff
@@ -22,6 +27,10 @@ jobs:
 
       - name: Generate terminal images with rich-codex
         uses: ewels/rich-codex@v1
+        env:
+          COLUMNS: 100
+          NFCORE_LINT_HIDE_PROGRESS: true
+          NFCORE_MODULES_LINT_HIDE_PROGRESS: true
```

Comment:
Hah, hadn't thought about this - that the env var would be different for different commands.

Ok so then this would be a vote for using an explicit `env="HIDE_PROGRESS"` on the commands so that we could have one env var that covers the entire tool (or using it on root in one place).

Again low priority. Basically this current setup is fine, might be nice to refactor one day (especially if we add more progress bars) but no needed.